azkaban-common/src/main/java/azkaban/executor/ExecutionRampDao.java

```diff
@@ -393,6 +411,28 @@ public void updateAction(final String tableName, Map<String, Object> actionData,
     }
 
     try {
+      if ("ramp".equalsIgnoreCase(tableName)) {
```

Comment:
This block will make sure the default value being added when changing record.
Let's say, if the client didn't checkin the lastUpdatedTime field for Ramp Table, the system will auto append the timestamp.

Reply:
Please add a comment in the code as well.